build/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.100-bionic AS build
```

Comment:
Should we delete the /samples/docker/Dockerfile now we have this one?

Reply:
Good catch. Deleted